README.md

```diff
@@ -127,7 +127,7 @@ Before you can run the app locally, you'll need to create your own OAuth 2.0 cre
 3. [Create a new project](https://cloud.google.com/apis/docs/getting-started#creating_a_google_project) in the Google Cloud Console and [enable](https://cloud.google.com/apis/docs/getting-started) the following APIs:
     i. [Google Analytics API](https://console.cloud.google.com/apis/api/analytics.googleapis.com/overview)
     ii. [Analytics Reporting API](https://console.cloud.google.com/apis/api/analyticsreporting.googleapis.com/overview)
-4. [Set up OAuth 2.0](https://support.google.com/cloud/answer/6158849) in your new project and create a client ID and make sure to add `localhost:4040` to the list of authorized domains.
+4. [Set up OAuth 2.0](https://support.google.com/cloud/answer/6158849) in your new project and create a client ID and make sure to add `localhost:4040` to the list of Authorised JavaScript origins.
```

Comment:
Thanks, I found this developer resource, which may be a better place to link to given that it uses "origins" instead of "domains": https://developers.google.com/identity/protocols/oauth2/javascript-implicit-flow#creatingcred

Also, nit: can you change the spelling of "Authorised" to "Authorized" (see [style guide](https://developers.google.com/style/translation#:~:text=We%20write%20our%20developer%20documentation%20in%20US%20English) for rationale)

Reply:
```suggestion
4. [Set up OAuth 2.0](https://developers.google.com/identity/protocols/oauth2/javascript-implicit-flow#creatingcred) in your new project and create a client ID and make sure to add `localhost:4040` to the list of Authorized JavaScript origins.
```

TIL that Google Cloud Console is regionalised between US and UK English depending on your settings! Here's what I saw, but changing my profile to US English does indeed use US spelling:

<img width="507" alt="image" src="https://user-images.githubusercontent.com/10931297/155898760-1753f1e2-cdbd-4b97-b9a7-73e1c104324a.png">

Reply:
Yeah, unfortunately we don't do the same for the developer docs.